lighthouse-core/test/results/sample_v2.json

```diff
@@ -3582,6 +3595,42 @@
         "duration": 100,
         "entryType": "measure"
       },
+      {
+        "startTime": 0,
+        "name": "lh:audit:first-contentful-paint-3g",
+        "duration": 100,
+        "entryType": "measure"
+      },
+      {
+        "startTime": 0,
+        "name": "lh:computed:FirstContentfulPaint",
```

Comment:
these duplicate keys are kind of a bummer

Reply:
yeah it is :(

I'm not sure it's worth sticking the options into the name though to differentiate either though :/

Reply:
> I'm not sure it's worth sticking the options into the name though to differentiate either though :/

yeah, will probably just confuse things without much benefit. If someone cares about that they'll be more likely to be looking at things in the timeline viewer anyways